tasks/markup-1.md

```diff
@@ -1,105 +1,73 @@
+	 - Using id for styles will be considered as a mistake, **minus 5 points**
 
-10. Footer должен прибит к низу. Прибитие значит, что если убрать контент на странице, кроме футера, то футер будет все равно внизу.
-**!оценка: 0-5**
+3. The logo should be implemented correctly. Don't forget to use *h1* tag on the page(**points - 0-5**);
 
-11. На сайте динамики может не быть, кроме событий hover. К ним можно и нужно применить стилевое оформление на ваше усмотрение, например при наведении курсором на элементы верхенго меню.
-**!оценка: 0-5**
+4. Navigation should be implemented with a *nav* tag. Clickable parts of the menu should be *links*(**points - 0-5**);
 
-Header можно делать фиксированным при прокрутке на фоне градиента, но доп. баллов за это не предусмотрено.
+5. Pay your attention to the fact that the background should be a graphic pattern, not just plain white(**points - 0-5**);
 
-Использовать можно любую сеточную технологию. Можно использовать font-awesome для иконок.
+6. Fonts. You can find all the font for free (the links will be provided). The fonts should be imported properly. Don't forget about the font-weight, it also should be correct(**points - 0-10**);
 
-Препроцессоры и постпроцессоры использовать не обязательно. Отдавать на выходе файлы типа .scss, .less нельзя. Также нельзя использовать bootstrap или подобную библиотеку. Все должно быть сделано самостоятельно. штраф **минус 40 баллов**.
+7. Requirements for the blocks and columns:
+	 - The case with more text than in the markup should be considered - just copy to the column bigger text(the position of the column or layout shouldn't change, limitation by overflow)(**points - 0-10**)
+	 - You should consider the case when one more column is added. This column should appear under the first one or in the center of the next line or hidden by overflow property. It should not break the layout by being positioned as the visible third element out of the parent's borders(**points - 0-5**);
+	 -  If you are going to use inline-block property, make sure that there is no free space between the elements(**points - 0-5**);
 
-## Полезные ссылки
+8. Images in *fine ingredients* section should be square or rectangle-shaped and should be processed by CSS tools(border, border-radius)(**points - 0-10**);
 
-[Презентации](https://github.com/rolling-scopes/front-end-course/wiki/Position.-Floats.-Semantics.-CSS3)
+9. The form should be implemented with a *form* tag and with the proper tags inside of it. The input fields should be validated with basic browser validation for Email, and Name, Email and Date fields should be required. One of the field's name is incorrect, you should fix it for your application(**points - 0-10**);
```

Comment:
The form should be implemented with a *form* tag and with proper tags inside of it. Add basic browser validation for Email input field. Name, Email and Date fields should be required. Please ignore typo on one of template's input fields, you should fix it in your application(**points - 0-10**);